documentation/STRUCTURE.md

```diff
@@ -0,0 +1,161 @@
+
+- [Group Metrics](https://github.com/fairlearn/fairlearn/blob/master/notebooks/Group%20Metrics.ipynb) - a great example for something that should be a user guide
+- [Grid Search for Binary Classification](https://github.com/fairlearn/fairlearn/blob/master/notebooks/Grid%20Search%20for%20Binary%20Classification.ipynb) - the purpose is mostly to show Grid Search's functionality; perhaps it may need to be trimmed down to the essentials about Grid Search
+- [Grid Search with Census Data](https://github.com/fairlearn/fairlearn/blob/master/notebooks/Grid%20Search%20with%20Census%20Data.ipynb) - similar to the previous notebook it covers Grid Search; we could leverage some of this for a user guide for Grid Search, or alternatively for the dashboard visualizations
+
+### Case studies
+
+The purpose of fairness case studies is to walk through an application of Fairlearn in detail. Any application of a fairness toolkit needs to be done with great care while taking into account an entire range of concerns due to the sociotechnical nature of fairness. The showcased case studies will provide the space to cover scenarios in depth. The focus is not only on showing example usage of the Fairlearn toolkit, but on how to approach fairness in ML in general. We may want to add scenario even if it contains only few of Fairlearn's capabilities, but it otherwise demonstrates a great example of how to build AI responsibly.
+
+All the case studies should be downloadable as Jupyter notebooks and/or Python source code, and be launchable in [Binder](https://mybinder.org/) and perhaps other platforms.
+
+Note: [scikit-learn](https://scikit-learn.org/dev/auto_examples/index.html) refers to these as "Examples".
+
+Of our current notebooks the following would be most closely aligned with this section:
+
+- [Binary Classification on COMPAS dataset](https://github.com/fairlearn/fairlearn/blob/master/notebooks/Binary%20Classification%20on%20COMPAS%20dataset.ipynb) - although we should perhaps consider removing it since it may not do justice to this complex setup
+- [Binary Classification with the UCI Credit-card Default Dataset](https://github.com/fairlearn/fairlearn/blob/master/notebooks/Binary%20Classification%20with%20the%20UCI%20Credit-card%20Default%20Dataset.ipynb)
+- [Mitigating Disparities in Ranking from Binary Data](https://github.com/fairlearn/fairlearn/blob/master/notebooks/Mitigating%20Disparities%20in%20Ranking%20from%20Binary%20Data.ipynb)
+
+### Community
+
+This page should provide basic information about how to reach out in order to ask questions, provide feedback, report bugs or request features. Similarly, it should be clear how people can reach out if they want to contribute to Fairlearn, and where they can find small items to get started. [This](https://scikit-learn.org/dev/developers/contributing.html) is an example of how scikit-learn handles it through a contribution guide that is somewhat similar to ours that we currently have in the repo.
```

Comment:
contributing guide and the community page are a bit different though. Ideally IMO you'd have a page which talks about your community and authors etc, and one for contributions. You can see the "community" section on the bottom of the page [here](https://scikit-learn.org/stable/)